near-contract-standards/src/fungible_token/core_impl.rs

```diff
@@ -48,7 +48,7 @@ pub trait FungibleTokenContract {
     /// Returns the total supply of the token in a decimal string representation.
     fn ft_total_supply(&self) -> U128;
 
-    /// Returns the balance of the account. If the account doesn't exist must returns `"0"`.
+    /// Returns the balance of the account. If the account doesn't exist must return `"0"`.
```

Comment:
```suggestion
    /// Returns the balance of the account. If the account doesn't exist `"0"` must be returned.
```

Reply:
```suggestion
    /// Returns the balance of the account. If the account doesn't exist, `"0"` must be returned.
```
Comma seems important to be correct